test/contract based tests/LosslessController.js

```diff
@@ -122,6 +122,39 @@ describe('Lossless Controller', () => {
     });
   });
 
+  describe('when pausing', () => {
+    it('should revert when not pause admin', async () => {
+      await expect(
+        env.lssController.connect(adr.regularUser1).pause(),
+      ).to.be.revertedWith('LSS: Must be pauseAdmin');
+    });
+
+    it('should not revert when not pause admin', async () => {
+      await expect(
+        env.lssController.connect(adr.lssPauseAdmin).pause(),
+      ).to.not.be.reverted;
+    });
+  });
+
+  describe('when unpausing', () => {
+    beforeEach(async () => {
+      await expect(
+        env.lssController.connect(adr.lssPauseAdmin).pause(),
+      ).to.not.be.reverted;
+    });
+    it('should revert when not pause admin', async () => {
+      await expect(
+        env.lssController.connect(adr.regularUser1).unpause(),
+      ).to.be.revertedWith('LSS: Must be pauseAdmin');
+    });
+
+    it('should not revert when not pause admin', async () => {
```

Comment:
I think it's supposed to be "<...> when pause admin"

Reply:
Addresed on 828ee96